src/main/java/org/mockito/ArgumentMatchers.java

```diff
@@ -474,9 +474,9 @@ public static String anyString() {
      * @see #isNull()
      * @see #isNull(Class)
      */
-    public static List anyList() {
+    public static <T> List<T> anyList() {
```

Comment:
The wildcard should be avoided in return types, especially this one since it's unbounded it forces the caller to deal with the wildcard (casting with mockito matchers). Also using `T` allows the Java 8 compiler to infer (capture) the wanted type.